apps/condo/domains/acquiring/utils/testSchema/index.js

```diff
@@ -430,6 +434,47 @@ async function makePayerAndPayments (receiptsAmount = 1) {
     }
 }
 
+/**
+ * Handles simplified payment case: 1 MultiPayment 1 Receipt 1 Payment
+ * As a resident pay for single billing receipt with specified <amount>,
+ * As an integrationClient complete payment
+ *
+ * Currently working with fees is not implemented
+ *
+ * @param {Object} residentClient
+ * @param {Object} integrationClient
+ * @param {string} serviceConsumerId
+ * @param {string} receiptId
+ * @param {Object} extra
+ * @return {Promise<{doneMultiPayment: ({data: *, errors: *}|*)}>}
+ */
+async function completeTestPayment(residentClient, integrationClient, serviceConsumerId, receiptId, extra = {}) {
+    const registerMultiPaymentPayload = {
+        consumerId: serviceConsumerId,
+        receipts: [{id: receiptId}],
+    }
+    const [ { multiPaymentId } ] = await registerMultiPaymentByTestClient(residentClient, registerMultiPaymentPayload)
+
+    // Acquiring integration makes payment and multiPayment done
+    const [ multiPayment ] = await MultiPayment.getAll(integrationClient, { id: multiPaymentId })
+    await updateTestPayment(integrationClient, multiPayment.payments[0].id, {
+        explicitFee: '0.0',
```

Comment:
May be pass fees as params?

Reply:
I think I lack good domain knowledge to do it properly, but added a todo in personal calendar